src/components/common/Card/Card.tsx

```diff
@@ -18,19 +19,33 @@ interface CardProps {
 
 const Card = ({ data }: CardProps): ReactElement => {
   const { title, description, date, image } = data;
+  const width = useWindowWidth();
+
+  const getImageWidthPerWidth = () => {
```

Comment:
이런 코드들은 함수로 처리하는 것보다 useMemo로 처리하는 것이 좋을 것 같아요.
칼럼에서 봤는데 넘겨주는 데이터를 함수로 처리하면 함수를 해석하고 실행하는 만큼 자원이 추가로 필요하다고 하더라고요.

아래와 같은 방법을 추천드려요.
```suggestion
  const getImageWidthPerWidth = useMemo(() => {
      if (width < BreakPoints.tablet) return 370;
      if (width < BreakPoints.mobile) return 280;
      return 384;
 }, [width, BreakPoints]);
```